ColabTool/Shared/PostFooter/index.jsx

```diff
@@ -58,13 +58,18 @@ const renderEdit = ({ onEditClick }) =>
 
 const renderApproval = ({
   onApproveClick,
-}) =>
-  <span className={style['post-button-last']}>
-    <span className={style['vertical-line']} />
-    <Button onClick={onApproveClick} noStyle>
-      <Text size={'small'} color={'blue'}>Approve</Text>
-    </Button>
-  </span>;
+  manager,
```

Comment:
Heya @hharnisc - love the idea of making this look cleaner but it looks like ESLint doesn't seem to like the ternary operator here. 🤔